src/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -279,6 +279,9 @@ public enum CopyPixelOperation
         SourcePaint = 15597702,
         Whiteness = 16711778,
     }
+#if netcoreapp
```

Comment:
NIT: if you wanted to avoid the define you could put these in a separate file.  EG: System.Drawing.Common.netcoreapp.cs
```c#
    [System.ComponentModel.TypeConverter("System.Drawing.FontConverter, System.Windows.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51")]
    public sealed partial class Font { }
    ...
```

I've done that in some other places.